pkg/coprocessor/coprocessor.pb.go

```diff
@@ -151,167 +151,167 @@ func init() {
 	proto.RegisterType((*Request)(nil), "coprocessor.Request")
 	proto.RegisterType((*Response)(nil), "coprocessor.Response")
 }
-func (m *KeyRange) Marshal() (data []byte, err error) {
+func (m *KeyRange) Marshal() (dAtA []byte, err error) {
```

Comment:
Yep, I think it's because that latest go protobuf has such (patch)[https://github.com/gogo/protobuf/commit/91cc2497504ffbaba9656e7d988ccfe41c54f6a9]